pages/getting-started/manage-git-repositories/workspace-encryption.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Workspace Encryption Guide
+description: How to use `plural crypto backups` to create, list, and restore workspace backups.
+---
+
+Out of the box, Plural encrypts all configuration in your Plural workspace and stores an encryption key on your local machine.
+This means that if anyone gets a copy of your Plural Git repository/workspace, all sensitive information will appear encrypted to them.
+
+If you are changing machines or collaborating with colleagues, you'll need to know how to create and restore from the workspace
+encryption backups.
+
+## Your encryption key
+
+When you initialize your Plural repository, Plural creates a directory in your home director called `.plural`. Within this directory
+is a file called `key`, which is your local encryption key. This key is required to decrypt any sensitive configuration within your
```

Comment:
is it documented somewhere how to do this? is that the crypto unlock command?

Reply:
it's autogenerated, but if you want to import one, you can use the `cat /path/to/key | plural crypto import` command

Reply:
added a note about this